src/index.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const state = {
+  tempValue: 80,
+  city: 'Seattle',
+  lat: 0,
+  lon: 0
+};
+
+const temperature = document.getElementById('tempValue');
+
+const increaseTemp = () => {
+  state.tempValue += 1;
+  temperature.textContent = state.tempValue + '°F';
```

Comment:
Using `+` to concatenate strings is totally fine, but you'll see template strings used more commonly, 
```javascript
`${state.tempValue}°F`
```